package.json

```
@@ -1,143 +1,7 @@
-    "@types/react": "^16.9.46",
-    "@types/react-dom": "^16.9.8",
-    "@types/styled-components": "^5.1.4",
-    "@types/styled-system": "^5.1.10",
-    "@types/styled-system__css": "^5.0.14",
-    "@types/unist": "^2.0.3",
-    "@types/webpack": "^4.41.21",
-    "@typescript-eslint/eslint-plugin": "^3.9.1",
-    "@typescript-eslint/parser": "^3.9.1",
-    "ajv": "^6.12.4",
-    "autoprefixer": "^10.0.0",
-    "babel-jest": "^26.3.0",
-    "babel-plugin-lodash": "^3.3.4",
-    "babel-plugin-module-resolver": "^4.0.0",
-    "babel-plugin-styled-components": "^1.11.1",
-    "chalk": "^4.1.0",
-    "cross-env": "^7.0.3",
-    "cypress": "^6.1.0",
-    "download": "^8.0.0",
-    "download-cli": "^1.1.1",
-    "eslint": "^7.7.0",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-plugin-react": "^7.20.6",
-    "eslint-plugin-react-hooks": "^4.1.0",
-    "globby": "^11.0.1",
-    "husky": "^4.2.5",
-    "identity-obj-proxy": "^3.0.0",
-    "jest": "^26.4.2",
-    "json-schema-to-typescript": "^9.1.1",
-    "lint-staged": "^10.2.11",
-    "lodash-webpack-plugin": "^0.11.5",
-    "meow": "^7.1.1",
-    "next-compose-plugins": "^2.2.0",
-    "next-transpile-modules": "^4.1.0",
-    "node-sass": "^4.14.1",
-    "postcss-flexbugs-fixes": "^4.2.1",
-    "postcss-preset-env": "^6.7.0",
-    "prettier": "^2.0.5",
-    "react-refresh": "^0.8.3",
-    "start-server-and-test": "^1.11.4",
-    "ts-jest": "^26.4.4",
-    "ts-node": "^9.0.0",
-    "typescript": "^4.1.2"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged",
-      "pre-push": "yarn run type-check"
-    }
-  },
-  "lint-staged": {
-    "*.@(ts|tsx)": [
-      "yarn lint",
-      "yarn format"
-    ]
-  },
-  "scripts": {
-    "analyze": "cross-env ANALYZE=true yarn build",
-    "export": "next export",
-    "dev": "next",
-    "build": "cross-env NEXT_TELEMETRY_DISABLED=1 && next build && next export",
-    "start": "next start",
-    "test": "jest --watch",
-    "test:ci": "jest --ci",
-    "format": "prettier --write **/*.{js,ts,tsx} *.{js,json,md,yml}",
-    "lint": "eslint .",
-    "type-check": "yarn compile",
-    "compile": "tsc --pretty --noEmit",
-    "validate-json": "ts-node --project src/tools/tsconfig.json src/tools/validator/validate-json.ts",
-    "validate-single": "ts-node --project src/tools/tsconfig.json src/tools/validator/validate-single.ts",
-    "generate-typescript": "ts-node --project src/tools/tsconfig.json src/tools/code-generation/generate-typescript.ts",
-    "e2e": "start-test dev 3000 cy:open",
-    "e2e:release": "yarn build && start-test 3000 cy:open",
-    "cy:open": "cypress open",
-    "cy:ci": "cypress run --headless --browser chrome --config-file ./cypress-ci.json",
-    "download": "download https://coronadashboard.rijksoverheid.nl/latest-data.zip -e -o public/"
-  },
-  "browserslist": [
-    "defaults",
-    "not ie < 11"
+  "workspaces": [
+    "corona-dashboard-cms",
+    "corona-dashboard-app"
   ]
```

As I mentioned before, it's cleaner to have all packages live in a dedicated directory, and then this config can be:

```suggestion
 "workspaces": {
    "packages": [
      "packages/*"
    ]
  },
```

---

I suspect also this is why workspaces isn't working, because the config structure differs